### ISSUE TITLE
Running the contributed command: 'lightSwitch.setDayTheme' failed.

### ISSUE THREAD

**Issue by curtisbowden** [timrodz/vscode-light-switch#22]
Running any lightSwitch commands fail, even though it appears to load correctly.

I've read through Readme.MD several times to be sure I didn't miss anything during the setup, screenshots show steps I took that brought me to the error.

<img width="809" alt="Screen Shot 2019-10-18 at 11 47 01 PM" src="https://user-images.githubusercontent.com/24552323/67139293-7697e300-f203-11e9-8b6a-770d7efc8031.png">

<img width="1156" alt="Screen Shot 2019-10-18 at 11 45 29 PM" src="https://user-images.githubusercontent.com/24552323/67139285-5700ba80-f203-11e9-9583-89a57b87cd04.png">

<img width="1158" alt="Screen Shot 2019-10-18 at 11 41 19 PM" src="https://user-images.githubusercontent.com/24552323/67139289-66800380-f203-11e9-94d7-26b6985c3d1c.png">

Also, running the test script shows the following in the debug console.

<img width="939" alt="Screen Shot 2019-10-18 at 11 46 40 PM" src="https://user-images.githubusercontent.com/24552323/67139327-dee6c480-f203-11e9-90ec-8814bb1fdc34.png">

**Comment by timrodz** [comment]
Hi @curtisbowden, are you running the extension through the debug menu?

<img width="138" alt="Screen Shot 2019-10-20 at 12 14 22 PM" src="https://user-images.githubusercontent.com/7434353/67152456-4b141600-f333-11e9-88d7-f31a55f3c2e3.png">

Is this configuration in your `launch.json`?

```json
{
  "name": "Run Extension",
  "type": "extensionHost",
  "request": "launch",
  "runtimeExecutable": "${execPath}",
  "args": ["--extensionDevelopmentPath=${workspaceFolder}"],
  "outFiles": ["${workspaceFolder}/out/**/*.js"],
  "preLaunchTask": "npm: watch"
},
```

**Comment by curtisbowden** [comment]
> Hi @curtisbowden, are you running the extension through the debug menu?
> 
> <img alt="Screen Shot 2019-10-20 at 12 14 22 PM" width="138" src="https://user-images.githubusercontent.com/7434353/67152456-4b141600-f333-11e9-88d7-f31a55f3c2e3.png">
> 
> Is this configuration in your `launch.json`?
> 
> ```json
> {
>   "name": "Run Extension",
>   "type": "extensionHost",
>   "request": "launch",
>   "runtimeExecutable": "${execPath}",
>   "args": ["--extensionDevelopmentPath=${workspaceFolder}"],
>   "outFiles": ["${workspaceFolder}/out/**/*.js"],
>   "preLaunchTask": "npm: watch"
> },
> ```

Hi @timrodz , my lanuch.jason looks similar to yours, although the formatting is a bit different, and there is also a similar section for the extension tests.

I've been attempting to run from the debug menu, though I also noticed a second debug menu on the status bar at the bottom of vscode, which I also tried.

<img width="712" alt="Screen Shot 2019-10-20 at 12 35 16 AM" src="https://user-images.githubusercontent.com/24552323/67156303-91d12400-f2d1-11e9-97c7-bd179e395194.png">

**Comment by timrodz** [comment]
Might be worth re-installing the project by directly cloning the repository. I really have no idea why it's not running.

**Comment by curtisbowden** [comment]
> Might be worth re-installing the project by directly cloning the repository. I really have no idea why it's not running.

Welp, I'm with you.. no idea why it wasn't working.  I cloned the main repo as you suggested and it worked fine.  The first time it worked, there was a few messages in the console I should have taken a screenshot.  They said something about deleting a cache entry somewhere in Library/Application Support... but I got distracted playing with the light switch plugin and the messages didn't show up the next time it ran.

I've deleted my repo and made a new fork.  I suspect we should archive this issue and watch for it to pop up again?
